### ISSUE TITLE
Putting a list of outreach channels and document it

### ISSUE THREAD

**Issue by realslimshanky** [pythonindia/inpycon2021-tasks#17]
The goal is to put all the mailing lists and other channels where news and announcements about the conference can be posted. CC @raukadah

**Comment by realslimshanky** [comment]
@raukadah Let's start this async and meet on a call whenever possible.
I've created [a google sheet](https://docs.google.com/spreadsheets/d/1co9BV2_WrVyJTbKez0CuLphSIz31rpCAlp0DlmLstbU/edit?usp=sharing) to list down all the mailing lists and also new columns can be used to manage post sent to the lists.

**Comment by realslimshanky** [comment]
> @raukadah Let's start this async and meet on a call whenever possible.
> I've created [a google sheet](https://docs.google.com/spreadsheets/d/1co9BV2_WrVyJTbKez0CuLphSIz31rpCAlp0DlmLstbU/edit?usp=sharing) to list down all the mailing lists and also new columns can be used to manage post sent to the lists.

**Comment by realslimshanky** [comment]
@raukadah I've updated the sheet with instructions about how to use it,
```
Guidelines to use:
--
1. Whenever there's a new email to send please add a new column and write the email URL/subject in row 1
2. Write your name in the row of the mailing list if you're taking responsibility for sending out the email to that mailing list
3. Once sent, replace your name with 'SENT'
```

**Comment by satyaakam** [comment]
more lists to be added 

1. https://groups.google.com/g/ilugcbe
2. https://groups.google.com/g/amity-linux-group
3. https://groups.google.com/g/nitdgplug
4. https://groups.google.com/g/jiitlug
5. https://groups.google.com/g/ml-india
6. https://groups.google.com/g/mumpy

there are some more will keep updating here

**Comment by satyaakam** [comment]
I see mails being sent to different mailing list as cross post , can we just put all the mailing  addresses in BCC ?

**Comment by sayanchowdhury** [comment]
I agree with @satyaakam. Rather than sending a separate email please cc, or bcc the mailing addresses. Though I would suggest to send a separate email to inpycon mailing list.

gist:
- main email to inpycon
- second email with all emails in cc and with [X-POST] in subject to the mailing list
- chair can send the email to PSF related mailing lists if needed.

**Comment by anistark** [comment]
Can we document these somewhere and close this issue?

@realslimshanky @abhishekmishragithub
